### ISSUE TITLE
Accessibility Report: Global Footer

### ISSUE THREAD

**Issue by sherakama** [SU-SWS/decanter#318]
Use the AMP tool to scan the [Global Footer](http://decanter.stanford.edu/section-components.html#kssref-components-global-footer) component for accessibility issues and document them to this ticket.

**Comment by sherakama** [comment]
AMP Report: https://stanford.levelaccess.net/public/reporting/view_report.php?report_id=283429

No violations found.

**Comment by sherakama** [comment]
Chrome Audit: 100

![Screen Shot 2019-03-13 at 9 10 10 PM](https://user-images.githubusercontent.com/550602/54330807-687e4600-45d4-11e9-8126-09d829e5e5fd.png)
